src/main/java/seedu/address/model/person/Person.java

```diff
@@ -96,6 +96,17 @@ public boolean isSamePerson(Person otherPerson) {
                 && otherPerson.getName().equals(getName());
     }
 
+    /**
+     * Checks if the given {@code prompt} is in the {@code financialPlans} and returns the email if true.
+     */
+    public String compareFinancialPlan(String prompt) {
+        FinancialPlan fp = new FinancialPlan(prompt);
+        if (financialPlans.contains(fp)) {
+            return this.email.toString();
+        }
```

Comment:
financialPlans is a HashSet and contains() method checks whether the hashcode of objects in the hashset and the object passed as argument corresponds.

Reply:
Perhaps it would be good to clarify this in the dev guide. Also, maybe "getEmailIfContainsFinancialPlan" would be a more indicative of the method's functionality.

Reply:
I have added description in the developers guide in the `Model` section, idk if this is correct.